src/runners/S3JUnitReporter.ts

```diff
@@ -0,0 +1,36 @@
+import { Callback } from 'aws-lambda';
+import { Credentials, S3 } from 'aws-sdk'
+import * as fs from 'fs';
+import * as moment from 'moment';
+
+export const S3JUnitReporter = (callback: Callback, s3BucketName:string, filePath: string, productName?:string) => {
+    return {
+      jasmineDone: async (result, done) => {
+        const reportContents = fs.readFileSync(filePath, 'utf8');
+        if(!productName) {
+          productName = 'complianceReport';
+        }
+        const now = moment().format('YYYY-MM-DD-HH-mm-ss');
+        const fileName = productName+'-'+ now + '.xml';

```

Comment:
prefer string interpolation here `${}`

Reply:
reads much better with interpolation. changed it